src/lwc/boutiqueAppointments/boutiqueAppointments.js

```diff
@@ -55,8 +61,20 @@ export default class BoutiqueAppointments extends NavigationMixin(LightningEleme
     @track pageSize = 5;
     @track totalRecountCount = 0;
     @track totalPage = 0;
+
+    @track status;
     isPageChanged = false;
 
+    get options() {
```

Comment:
MAJOR
Never not hardcode field values as someone could change it in CRM and that change will never be reflected here.
Use OOTB methods: https://developer.salesforce.com/docs/component-library/documentation/en/lwc/lwc.reference_wire_adapters_picklist_values_record